tests/resources/dataplex_gcs_metadata_expected_configs.json

```diff
@@ -0,0 +1,82 @@
+{
+    "rule_binding_id":"<dataplex_rule_binding_id>",
```

Comment:
please pass this json through a pretty printer. there should be a space after the colon.

Reply:
done